src/datasources/db/postgres-database.migrator.spec.ts

```diff
@@ -0,0 +1,318 @@
+import { dbFactory } from '@/__tests__/db.factory';
+import postgres from 'postgres';
+import path from 'node:path';
+import fs from 'node:fs';
+import { PostgresDatabaseMigrator } from '@/datasources/db/postgres-database.migrator';
+
+const folder = path.join(__dirname, 'migrations');
+const migrations: Array<{
+  name: string;
+  file: { name: string; contents: string };
+}> = [
+  {
+    name: '00001_initial',
+    file: {
+      name: 'index.sql',
+      contents: `drop table if exists test;
+                 create table test (
+                   a text,
+                   b int
+                 );
+
+                 insert into test (a, b) values ('hello', 1337);`,
+    },
+  },
+  {
+    name: '00002_update',
+    file: {
+      name: 'index.js',
+      contents: `module.exports = async function(sql) {
+                  await sql\`
+                    alter table test add column c timestamp with time zone
+                  \`
+
+                  await sql\`
+                    insert into test (a, b, c) values ('world', 69420, ${'${new Date()}'})
+                  \`
+                }`,
+    },
+  },
+  {
+    name: '00003_delete',
+    file: {
+      name: 'index.sql',
+      contents: 'drop table test;',
+    },
+  },
+];
+
+describe('PostgresDatabaseMigrator tests', () => {
+  let sql: postgres.Sql;
+  let target: PostgresDatabaseMigrator;
+
+  beforeEach(() => {
+    sql = dbFactory();
+    target = new PostgresDatabaseMigrator(sql);
+  });
+
+  afterEach(async () => {
+    // Drop example table after each test
+    await sql`drop table if exists tests`;
```

Comment:
```suggestion
    await sql`drop table if exists test`;
```

Reply:
Fixed in f994465bd3345e2870f0ce090c90c6b516bbcead.